CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 Change Log
 ==========
+
+Version 0.9.1 *(19th September 2022)*
+-------------------------------------------
+- Supports CleverTap Android SDK [v4.6.2](https://github.com/CleverTap/clevertap-android-sdk/blob/corev4.6.2_ptv1.0.5/CHANGELOG.md) and associated enhancements
+
```

Comment:
@root-ansh-ct We should mention that this supports XPS changes as well. With the link to XPS Changelog